libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -586,29 +476,25 @@ private void addBottomSheetListener() {
   }
 
   private void initializeClickListeners() {
```

Comment:
We can update the naming here to better describe the button setup functions it's calling